webpack.config.js

```diff
@@ -0,0 +1,22 @@
+var path = require('path');
+var SRC_DIR = path.join(__dirname, '/react-client/src');
+var DIST_DIR = path.join(__dirname, '/react-client/dist');
```

Comment:
Check to make sure file paths are matching your current folder structure

Reply:
Thanks for catching this :) I just updated